orchagent/flexcounterorch.cpp

```diff
@@ -86,10 +86,11 @@ void FlexCounterOrch::doTask(Consumer &consumer)
 
         if (op == SET_COMMAND)
         {
-            auto it = std::find(std::begin(data), std::end(data), FieldValueTuple(FLEX_COUNTER_DELAY_STATUS_FIELD, "true"));
+            auto delay_it = std::find(std::begin(data), std::end(data), FieldValueTuple(FLEX_COUNTER_DELAY_STATUS_FIELD, "true"));
```

Comment:
I think our code style is to use camelCaseStyle for local variables

Reply:
Fixed.